operator/README.md

```diff
@@ -1,2 +1,92 @@
-# pmem-csi-operator
-Operator for PMEM-CSI
+# PMEM-CSI-Operator
+
+Kubernetes operator for [PMEM-CSI](https://github.com/intel/pmem-csi) driver deployment. Using this operator one can easily deploy and manage PMEM-CSI driver in a Kubernetes cluster.
+
+The PMEM-CSI operator is based on the CoreOS [operator-sdk](https://github.com/operator-framework/operator-sdk) tools and APIs.
+
+## PMEM-CSI Deployment custom resource definition(CRD)
+
+Current PMEM-CSI Deployment object supports below API:
+
+### Deployment
+
+|Field | type | Description |
+|---|---|---|
+| apiVersion | string  | `pmem-csi.intel.com/v1alpha1`|
+| kind | string | `Deployment`|
+| metaData | [ObjectMeta](https://git.k8s.io/community/contributors/devel/api-conventions.md#spec-and-status) | Standard objects metadata. |
+| spec | [DeploymentSpec](#deployment-spec) | Specification of the desired behavior of the deployment |
+
+### DeployemtSpec
+
+|Field | type | Description | Default Value |
+|---|---|---|---|
+| image | ImageInfo | PMEM-CSI driver container image information | `intel/pmem-csi-driver:canary` |
+| logLevel | integer | PMEM-CSI driver logging level | 3 |
+| controller | CotrollerDriver | configuration of PMEM-CSI driver running as controller | |
+| node | NodeDriver | configuration of PMEM-CSI driver running at compute node | |
+
+### ControllerDriver
+
+|Field | type | Description | Default Value |
+|---|---|---|---|
+| provisionerImager | ImageInfo | CSI provisioner container image information | `quay.io/k8scsi/csi-provisioner:v1.2.1` |
+| resources | [ResourceRequirements](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.12/#resourcerequirements-v1-core) | describes the compute resource requirements for controller pod |
```

Comment:
Instead of blank "default value", put "none"?

We probably should change that (see https://github.com/intel/pmem-csi/issues/261), but not now.

Reply:
Current code is using the below defaults, which should be reviewed:

```
	defaultControllerResourceCPU    = int64(100) // MilliSeconds
	defaultControllerResourceMemory = int64(250) // MB
	defaultNodeResourceCPU          = int64(100) // MilliSeconds
	defaultNodeResourceMemory       = int64(250) // MB
```

I guess I should update the `Default Value` column with these values.